packages/config/src/android/Updates.ts

```diff
@@ -0,0 +1,103 @@
+import { ExpoConfig } from '../Config.types';
+import { Document } from './Manifest';
+
+export function getUpdateUrl(config: ExpoConfig, username: string | null) {
```

Comment:
is there a way we could determine if the user is self-hosting their updates? Maybe we could add an app.json field for this, otherwise that is the one setting they would need to configure manually in AndroidManifest and Info.plist, right?

Reply:
It would be great to add an app.json field or something for this -- another reason that would be nice is that you wouldn't have to provide the url every time you run `expo export`.

That said, `expo export` (same as `expo publish`) will also set the updateUrl field in the native project configs. So the only time it matters (and the developer would have to set it manually) is for builds made right after ejecting but before exporting.

Reply:
Okay, good to have this :
> It would be great to add an app.json field or something for this 

as a todo to improve the workflow. 
 

> the only time it matters is for builds made right after ejecting but before exporting

I also doubt many users would do this (eject, not export, build), so (i think) this is a very narrow edge case we're considering. Too small to warrant logging any warning, since it wouldn't be applicable to most folks. What do you think, is it worth us prioritizing adding the `exportUrl` key soon?

Reply:
opened this issue to track https://github.com/expo/expo-cli/issues/2285